src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
```

Comment:
@wangxdflight Exactly.
Well tty-1 generally does work but not with our setup since it is already in use because it has I2C. All four ports are in use with GPS, RC, distance sensor and motors. Why do you need to use tty-1?